src/components/order-service/order-service-review.html

```diff
@@ -27,33 +31,41 @@ <h3 class="review-message">
     </div>
   </div>
   <div ng-if="$ctrl.orderComplete">
-    <div class="review-status">
+    <div class="results-status">
       <span class="pficon pficon-ok" aria-hidden="true"></span>
       <span class="sr-only">Success</span>
-      <h3 class="review-message">
-        <span>
-          <strong>{{$ctrl.serviceInstance.metadata.name}}</strong> has been added to <strong>{{$ctrl.projectDisplayName}}</strong> successfully
-        </span>
+      <h3 class="results-message">
+        <strong>{{$ctrl.serviceInstance.metadata.name}}</strong> has been added to <strong>{{$ctrl.projectDisplayName}}</strong> successfully.
```

Comment:
`$ctrl.serviceClass.name` so that it's consistent with the pending message?

Reply:
I debated that, but the existing was $ctrl.serviceInstance.metadata.name, and binding success uses $ctrl.serviceInstance.metadata.name, so it seemed to make sense to keep it the same so the results step had parallel names.  Yea/nay?

![screen shot 2017-09-26 at 11 31 00 am](https://user-images.githubusercontent.com/895728/30978526-cad90c24-a448-11e7-9a19-c6c717980c3b.PNG)